llms/mlx_lm/tuner/trainer.py

```diff
@@ -127,18 +138,24 @@ def train(
     val_dataset,
     args: TrainingArgs = TrainingArgs(),
     loss: callable = default_loss,
-    iterate_batches: callable = iterate_batches
+    iterate_batches: callable = iterate_batches,
+    training_callback=None,
 ):
+    print(f"Starting training..., iters: {args.iters}")
+
     # Create checkpoints directory if it does not exist
     if not os.path.exists("checkpoints"):
         os.makedirs("checkpoints")
 
+    if training_callback is not None:
+        training_callback.on_train_begin(args.iters)
```

Comment:
This looks like a bug?

Reply:
Yes, this was an bug I introduced when rebasing the main branch and it's fixed now.